libraries/AP_SerialManager/AP_SerialManager.h

```diff
@@ -122,6 +125,13 @@ class AP_SerialManager {
     /* Do not allow copies */
     CLASS_NO_COPY(AP_SerialManager);
 
+    enum SerialPhysical {
+        SerialPhysical_Disable = -1,
+        SerialPhysical_USART = 0,
+        SerialPhysical_IP = 1,
```

Comment:
I wanna hear more about ```SerialPhysical_IP``` :)

Reply:
🤫 Shhhh.... you are ruining the Christmas surprise.